gc/base/standard/HeapWalker.cpp

```diff
@@ -93,8 +94,10 @@ heapWalkerObjectSlotDo(OMR_VMThread *omrVMThread, MM_HeapRegionDescriptor *regio
 	 * OK. We should simplify this code once classes-on-heap is enabled
```

Comment:
Done (comment cleanup). Good point about indirect objects. Scavenger needs to deal with this as well, and also allows for only one indirect (unreachable from reference graph) object per reachable heap object. I opted for this to avoid requiring a second type of object scanner/iterator (eg, getIndirectObjectIterator(omrobjectptr_t). Perhaps we need one but current model supports Java/example and I've not heard of any new requirements from other languages/frameworks.